src/SimpleAzure.Storage.HybridQueue/HybridMessage.cs

```diff
@@ -1,3 +1,3 @@
 namespace WorldDomination.SimpleAzure.Storage.HybridQueues;
 
-public record class HybridMessage<T>(T? Content, string MessageId, string PopeReceipt, Guid? BlobId);
+public sealed record HybridMessage<T>(T Content, string MessageId, string PopeReceipt, Guid? BlobId);

```

Comment:
why `sealed` ? what does this gain? Why would you not want other people to use this and inherit from this (if my memory of `sealed` is correct'ish)

Reply:
It's a minor perf gain. But also, nothing in this class is remotely designed for inheritance. Nothing is virtual, so what could possibly be gained from inheriting?

Reply:
VERY interesting ideology going on here. I .. might just go with this ....